### ISSUE TITLE
Collection.json/workspace-schematics.json missing from tools/schematics

### ISSUE THREAD

**Issue by KevinTaleb** [nrwl/nx#1232]
# Prerequisites

- [x ] I am running the latest version
- [x ] I checked the documentation and found no answer
- [x ] I checked to make sure that this issue has not already been filed
- [x ] I'm reporting the issue to the correct repository (not related to Angular, AngularCLI or any dependency)

Not sure if this is intended but it seems like the `collection.json` is missing from `tools/schematics`, therefore I'm unable to pass aliases to my schematics.
I found the `workspace-schematics.json` in the final build which is one lined and even if I add the alias there, it will be overwrite after building a new schematic.

I've also tried to create either a `collection.json` and `workspace-schematics.json` in `tools/schematics` but in the former case it will be ignored and in the latter my json is being overwritten.

So basically, I want to be able to edit a somewhat `collection.json` in order to add aliases, description etc.

**Comment by FrozenPandaz** [comment]
At the time, it was generated to make it easier for developers to add new schematics.

I think we could easily update a `collection.json` when we generate `workspace-schematics`.

Would people prefer this way?

**Comment by hadrien-toma** [comment]
I would love to be able to configure a `collection.json` file too: :100: !

**Comment by GreatHawkeye** [comment]
Definitely! I think developpers would have more flexibility and power over their schematics. Modifying `factory` and `aliases` are a must have I believe. :)

**Comment by Oussama-Mejid** [comment]
> At the time, it was generated to make it easier for developers to add new schematics.
> 
> I think we could easily update a `collection.json` when we generate `workspace-schematics`.
> 
> Would people prefer this way?

This would be a great feature  :+1:

**Comment by Hotell** [comment]
`collection.json` is a must and pre-requirement to be able to write tests for custom workspace schematic :)

```ts
// no collection no fun! 😆
const testRunner = new SchematicTestRunner('schematics', collectionPath);
```

![image](https://user-images.githubusercontent.com/1223799/74744180-13c26100-5262-11ea-833e-67d08c6215ac.png)

**Comment by FrozenPandaz** [comment]
Hi, sorry about this.

This was mislabeled as stale. We are testing ways to mark _not reproducible_ issues as stale so that we can focus on actionable items but our initial experiment was too broad and unintentionally labeled this issue as stale.

**Comment by Markus-Ende** [comment]
Hi, was this mislabeled as stale again?

**Comment by hadrien-toma** [comment]
@FrozenPandaz can we re-open this one please? In my point of view it is really important to be able to classify our generators and so having a deep file system would be really appreciated...
